deepfence_server/handler/scan_reports.go

```diff
@@ -803,17 +803,20 @@ func (h *Handler) stopScan(w http.ResponseWriter, r *http.Request, tag string) {
 	}
 
 	if req.ScanType == "CloudComplianceScan" {
-		log.Info().Msgf("CloudComplianceScan request, type: %s, scanid: %s",
-			req.ScanType, req.ScanID)
-		err = reporters_scan.StopCloudComplianceScan(r.Context(), req.ScanType, req.ScanID)
+		tag = "StopCloudComplianceScan"
```

Comment:
nit: `tag` is not needed, it's redundant information.
We have the `ScanType` and we have the function call printed in our logs.

Reply:
I am using it in the error log below.